pkg/apis/core/types.go

```diff
@@ -229,7 +229,7 @@ type PersistentVolumeSource struct {
 	// More info: https://releases.k8s.io/HEAD/examples/volumes/storageos/README.md
 	// +optional
 	StorageOS *StorageOSPersistentVolumeSource
-	// CSI (Container Storage Interface) represents storage that handled by an external CSI driver (Beta feature).
```

Comment:
seems like this should be changed when the feature gate limiting this field (CSIPersistentVolume) is promoted out of beta